inspection/data/results.log

```diff
@@ -0,0 +1,70 @@
+{'page_j': 1, 'test': 'equality', 'page_i': 1, 'result': 'pass', 'case': 'DefaultTest'}
```

Comment:
Does that mean this `.log` file is read as input by some other code? Ah, I see https://github.com/openstax/test-automation/pull/14/files/81f424bc811cf846cb71cdc8a0136d6b6a82edae#diff-da7b871ee3cb247829bf633e27767b29R185

Ok, I was just worried that if this file isn't (eventually) `.gitignore`'d then it'll clutter up the repo once larger PDF's are compared.

okies 😄